src/Smalot/PdfParser/Config.php

```diff
@@ -56,22 +56,22 @@ public function setFontSpaceLimit($value)
         $this->fontSpaceLimit = $value;
     }
 
-    public function getPdfWhitespaces(): string
+    public function getPdfWhitespaces()
     {
         return $this->pdfWhitespaces;
     }
 
-    public function setPdfWhitespaces(string $pdfWhitespaces): void
+    public function setPdfWhitespaces(string $pdfWhitespaces)
     {
         $this->pdfWhitespaces = $pdfWhitespaces;
     }
 
-    public function getPdfWhitespacesRegex(): string
+    public function getPdfWhitespacesRegex()
     {
         return $this->pdfWhitespacesRegex;
     }
 
-    public function setPdfWhitespacesRegex(string $pdfWhitespacesRegex): void
```

Comment:
We merged in #383 lately, which means we only support 7.1+. Support for earlier PHP versions is not required anymore. I like these explicit return values, can you re-add them please?